### PR TITLE
Fixed typos in README.md

### PR DESCRIPTION
**Pull request by vsharika1**
(3) "a kit to" -> "A kit to"
(3) "demo apps and test ideas" -> "demo applications and test cases"
(13) "Why make this?" -> "Why did we develop this?"
(15) "a kit to build quick demo apps and test ideas (chumming the water so to speak )" -> "To build quick demo applications and test cases."
(17) "routing" -> "Routing"
(18) "who" -> "Who"
(19) "simple" -> "Simple"
(30) "what's a chumbucket" -> "What is a chumbucket?"
(32) "a bucket" -> "A bucket"

### CODE REVIEW

**Comment by stagfoo** [comment]
Hey @vsharika1 Thanks for the readme updates, happy to merge this if you remove the updates replacing  `idea` with `case`:
- **a test case** is normally a well thought out implementation of a narrow concept or feature
- **testing an idea** is the process of exploring a broad concept which will change over time.
the point of the kit is to create some fast and loose

**Comment by vsharika1** [comment]
I did so. Please, let me know if the changes are visible. @stagfoo

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <p align="center"><img width="300px" src=".readme/logo.png" />
 </p>
-<p align="center">a kit to build quick demo apps and test ideas (chumming the water so to speak )</p>
+<p align="center">A kit to build quick demo applications and test cases (chumming the water so to speak )</p>
```
Comment (stagfoo) [inline README.md]:
```suggestion
<p align="center">A kit to build quick demo applications and test ideas (chumming the water so to speak )</p>
```

Comment (stagfoo) [reply README.md]: @vsharika1 here also 👼

**Comment by vsharika1** [comment]
Thanks for the reminder mate.

**Comment by stagfoo** [comment]
Thank you for the PR ❤️

**Pull request merged**